netplan_cli/cli/state_diff.py

```diff
@@ -442,6 +486,62 @@ def _normalize_routes(self, routes: set) -> set:
 
         return new_routes_set
 
+    def _normalize_gateways(self, config: dict) -> AbstractSet[NetplanRoute]:
```

Comment:
thought: Wow, that's lots of code/logic for those deprecated `gateway4/6` fields.. But now that we have it we should keep it ;-)